src/components/scene/embedded.js

```diff
@@ -0,0 +1,14 @@
+var registerComponent = require('../../core/component').registerComponent;
+
+module.exports.Component = registerComponent('embedded', {
+  schema: { default: true },
+
+  update: function () {
+    var sceneEl = this.el;
+    if (this.data === true) {
```

Comment:
the less you rely on implicit type coercion the more robust and less error prone is your code.